app/static/app.js

```diff
@@ -153,6 +154,16 @@ for (let i = 0; i < labels.length; i++) {
   title.className = "modal_title";
   title.innerHTML = label.title;
 
+  if (label.title_annotation) {
+    const title_annotation = document.createElement("span");
+    title_annotation.className = "modal_title_annotation";
+    title_annotation.innerHTML = label.title_annotation;
+    title.innerHTML = title.innerHTML.replace(
+      /<\/p>/g,
+      `${title_annotation.outerHTML}</p>`
+    );
```

Comment:
I would expect a nonbreaking space here, so we don't get `facsimile` on its own line?

Reply:
@cogat I don't think there is a space separator - the space/padding is handled in css.